compose-builder/add-taf-app-services.yml

```diff
@@ -22,9 +22,6 @@ services:
       - 59705:59705/tcp
     container_name: app-functional-tests
     hostname: app-functional-tests
-    env_file:
-      - common.env
-      - as-common.env
```

Comment:
Lost env_file `common-non-security.env`?

Reply:
Fix here and below. THX!